devtools/db_import/main.py

```diff
@@ -23,7 +23,7 @@
 with open(SCRIPT_DIR / "config.yml") as fd:
   config_file = yaml.safe_load(fd)
 
-config = config_file["cloud_functions"][os.environ["config_name"]]
+config = config_file["pipelines"][os.environ["config_name"]]
```

Comment:
Maybe we can also define it as a constant somewhere?

Reply:
Yes. Good idea. I put the constant in a new module `config` which I will add more stuff in a subsequent PR anyway.

I had to rename some variables though because of name clashing.